# Our build system exports SHELLOPTS which makes testing fail We could have set
# "unexport SHELLOPTS" in our Makefile, but maybe this will be accepted by
# community?
#
# Reported to bug-bash@gnu.org

--- tests/run-all	2016-11-28 00:46:51.812583704 -0800
+++ tests/run-all	2016-11-28 00:45:20.302845042 -0800
@@ -17,7 +17,7 @@ export PATH
 # unset BASH_ENV only if it is set
 [ "${BASH_ENV+set}" = "set" ] && unset BASH_ENV
 # ditto for SHELLOPTS
-#[ "${SHELLOPTS+set}" = "set" ] && unset SHELLOPTS
+[ "${SHELLOPTS+set}" = "set" ] && unset SHELLOPTS
 
 : ${THIS_SH:=../bash}
 export THIS_SH
